deploy/core/001_bridge.ts

```diff
@@ -11,7 +12,9 @@ const deployFunc: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 
   await deploy('Bridge', {
     from: deployer,
-    log: true
+    log: true,
+    gasLimit: BigNumber.from('42032000'), // Mandala
+    gasPrice: BigNumber.from('200786445289'), // Mandala
```

Comment:
These changes are temporary and only for deployment. Should not be required on Karura or Acala main nets.